### PR TITLE
create page Jars

### PR DESCRIPTION
**Pull request by Dmytro-Foliushniak**

### CODE REVIEW

File: frontend/mono_frontend/src/components/Jar.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
+export const Jar = () => {
+	return (
+		<>
+			<svg
+				id='Layer_1'
+				data-name='Layer 1'
+				xmlns='http://www.w3.org/2000/svg'
+				viewBox='0 0 450 700'
+				// className='jar_icon'
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/components/Jar.jsx:9]: leftover

File: frontend/mono_frontend/src/App.css [hunk 2]
```diff
@@ -52,7 +52,8 @@
 
 body {
     margin: 0;
-    font-family: Arial, sans-serif;
+    font: 400 16px/22px 'Inter', sans-serif;
+    /*font-family:  Inter, sans-serif;*/
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/App.css:56]: Leftover

File: frontend/mono_frontend/src/components/Jar.jsx (hunk 1 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/components/Jar.jsx]: Done

File: frontend/mono_frontend/src/App.css (hunk 2 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/App.css]: Done

File: frontend/mono_frontend/src/App.js [hunk 3]
```diff
@@ -7,6 +7,8 @@ import AuthenticatedContent from './pages/AuthenticatedContent/AuthenticatedCont
 import Root from './pages/Root'
 import { checkAuthLoader } from './utils/auth'
 import { CardsList } from './pages/CardsList/CardsList'
+import { JarDetails } from './pages/JarDetails/JarDetails'
+import { JarsList } from './pages/JarsList/JarsList'
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/App.js:11]: seems not used

File: frontend/mono_frontend/src/components/JarsListView.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	return (
+		<div className='jars'>
+			<div className='jars__box'>
+				<h1 className='jars__title'>Jars</h1>
+				<p className='jars__textMoney'>
+					Total balance in all your JARS: {totalSum} ₴
+				</p>
+				<ul className='jars__list'>
+					{jarsData.map(jar => (
+						<li className='jars__item'>
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/components/JarsListView.jsx:26]: IMO we can store single jar list item in separate component to reduce some visual pressure

File: frontend/mono_frontend/src/components/__tests__/JarsListView.test.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { JarsListView } from '../JarsListView'
+import { MemoryRouter } from 'react-router-dom'
+
+describe('JarsListView with jars data', () => {
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/components/__tests__/JarsListView.test.js:6]: Lets set one naming convetions of tests in frontend. If you have any documented best practices or suggestions - lets store it in project documents

зображення

File: frontend/mono_frontend/src/pages/JarsList/JarsList.css [hunk 6]
```diff
@@ -0,0 +1,108 @@
+.jars__box {
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/pages/JarsList/JarsList.css:1]: can we fix some point in Jar List css:
* Create some space (margin) between vertical list items
* Make sure, we don't scroll horisontally (at least on Web) our jar list. Ideally - mobile too

File: frontend/mono_frontend/src/pages/JarsList/JarsList.jsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+	const token = useRouteLoaderData('token')
+	const [jarsData, setJarsData] = useState(null)
+
+	useEffect(() => {
+		const fetchData = async function () {
+			try {
+				const access = await getJars(token)
+				setJarsData(access)
+			} catch (error) {
+				console.error('Error fetching card data:', error)
```
Comment (Iron-Cow) [inline frontend/mono_frontend/src/pages/JarsList/JarsList.jsx:37]: suggestion: we have [PopUpManager](https://github.com/Iron-Cow/MonoProject/blob/master/frontend/mono_frontend/src/components/PopUpManager.jsx) and can use it to visualise some issues for user. 

Here is some [example](https://github.com/Iron-Cow/MonoProject/blob/a772f49d2b50359a64df66c02f8b48a5656f9ac0/frontend/mono_frontend/src/pages/LoginPage.jsx#L75)

File: frontend/mono_frontend/src/App.js (hunk 3 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/App.js:10]: Done

File: frontend/mono_frontend/src/components/JarsListView.jsx (hunk 4 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/components/JarsListView.jsx]: Done

File: frontend/mono_frontend/src/components/__tests__/JarsListView.test.js (hunk 5 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/components/__tests__/JarsListView.test.js:6]: Done

File: frontend/mono_frontend/src/pages/JarsList/JarsList.css (hunk 6 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/pages/JarsList/JarsList.css:1]: Done

File: frontend/mono_frontend/src/pages/JarsList/JarsList.jsx (hunk 7 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/pages/JarsList/JarsList.jsx]: Done

File: frontend/mono_frontend/src/pages/JarsList/JarsList.css (hunk 6 above)
Comment (Iron-Cow) [reply frontend/mono_frontend/src/pages/JarsList/JarsList.css:1]: It is still without margin:

зображення



Yes, you removed scrollbar, but items still out of reach:

зображення

**Review by Iron-Cow — CHANGES_REQUESTED** [review]

File: frontend/mono_frontend/src/pages/JarsList/JarsList.css (hunk 6 above)
Comment (Dmytro-Foliushniak) [reply frontend/mono_frontend/src/pages/JarsList/JarsList.css:1]: Done

**Review by Iron-Cow — APPROVED** [review]
👍

**Review by Iron-Cow — APPROVED** [review]

**Pull request merged**